web/js/components/map/ol-measure-tool.js

```diff
@@ -49,19 +48,42 @@ let init = false;
 const allMeasurements = {};
 const vectorLayers = {};
 const sources = {};
+let draw;
 
 /**
  * A component to add measurement functionality to the OL map
  */
 function OlMeasureTool (props) {
-  let draw;
   let drawChangeListener;
   let rightClickListener;
   let twoFingerTouchListener;
 
   const {
     map, olMap, crs, unitOfMeasure, toggleMeasureActive, updateMeasurements, projections, proj,
   } = props;
+  const previousCrs = usePrevious(crs);
+
+  function usePrevious(data) {
+    const ref = useRef();
+    useEffect(() => {
+      ref.current = data;
+    }, [data]);
+    return ref.current;
+  }
+
+  // Listen for a projection change
+  useEffect(() => {
+    if (olMap != null) {
+      // Terminate draw for the *previous* projection
+      const geographyToTerminate = getProjectionFromCrs(previousCrs);
```

Comment:
Not a big deal but, the function probably isn't necessary since you can just as easily get the projection name from the crs in a single line by referencing the object directly here:
```
const projFromCrs = {
 'EPSG:4326': 'geographic',
 ...
}
const projToTerminate = projFromCrs[previousCrs];
```